src/client/Microsoft.Identity.Client/TokenCacheNotificationArgs.cs

```diff
@@ -255,5 +256,13 @@ public TokenCacheNotificationArgs(    // only use this constructor in product co
         /// Cache Details contains the details of L1/ L2 cache for telemetry logging.
         /// </summary>
         public TelemetryData TelemetryData { get; }
+
+        /// <summary>
+        /// Determines if the client application authentication is an FMI node under an RMA.
+        /// </summary>
+        public bool IsFmiClientNode 
```

Comment:
I think @jmprieur provided feedback that this is too specific for FMI and a more general purpose mechanism can be used. 

In my opinion, Id.Web Cache project needs to throw a very clear error message of why distributed cache cannot be used with FMI. 

Maybe we want to expose a `string NoDistributedCacheUseReason` with the error message?

Reply:
Would be a good idea.
The presence of the message would actually act as the bool?

Reply:
Sure, I can do that